.github/workflows/ci.yml

```diff
@@ -40,18 +40,20 @@ jobs:
                 args: analyse
 
     ci:
-        name: Test PHP ${{ matrix.php-version }} ${{ matrix.name }}
+        name: Test PHP ${{ matrix.php-version }} & Symfony ${{ matrix.symfony-version }}
         runs-on: ubuntu-latest
+        env:
+            SYMFONY_REQUIRE: ${{ matrix.symfony-version }}
         strategy:
             fail-fast: false
             matrix:
                 php-version: ['7.4', '8.0']
+                symfony-version: ['4.4.*', '5.3.*']
                 composer-flags: ['']
                 name: ['']
                 include:
-                    - php-version: 7.4
-                      composer-flags: '--prefer-lowest'
-                      name: '(prefer lowest dependencies)'
```

Comment:
Why did you remove this test ? it's supper important for us to ensure BC with older SF version